src/index.d.ts

```diff
@@ -4829,55 +4829,94 @@ declare namespace admin.projectManagement {
   }
 
   /**
-   * Metadata about a Firebase Android app.
+   * Metadata about a Firebase app.
    */
-  interface AndroidAppMetadata {
+  interface AppMetadata {
 
     /**
-     * The fully-qualified resource name that identifies this app.
-     *
-     * This is useful when manually constructing requests for Firebase's public API.
+     * The globally unique, Firebase-assigned identifier of the app.
      *
      * @example
      * ```javascript
-     * var resourceName = androidAppMetadata.resourceName;
+     * var appId = appMetadata.appId;
      * ```
      */
-    resourceName: string;
+    appId: string;
 
     /**
-     * The globally unique, Firebase-assigned identifier of the app.
+     * The optional user-assigned display name of the app.
      *
      * @example
      * ```javascript
-     * var appId = androidAppMetadata.appId;
+     * var displayName = appMetadata.displayName;
      * ```
      */
-    appId: string;
+    displayName?: string;
```

Comment:
@nbegley, @hiranya911.  What is the practice? `displayName?: string` is the same as `displayName: string | undefined` instead of `string | null`

Reply:
`displayName?: string` allows an object using that interface to be created without specifying the `displayName`, so I'd prefer that.

Example:

```typescript
interface MyInterface1 {
    myField?: string;
}

// Okay.
let myObj1_1: MyInterface1 = {};

// Okay.
let myObj1_2: MyInterface1 = {
    myField: undefined,
};

// Okay.
let myObj1_3: MyInterface1 = {
    myField: null,
};

// Okay.
let myObj1_4: MyInterface1 = {
    myField: "",
};

interface MyInterface2 {
    myField: string | null;
}

// Property 'myField' is missing in type '{}' but required in type 'MyInterface2'.
let myObj2_1: MyInterface2 = {};

// Okay.
let myObj2_2: MyInterface2 = {
    myField: undefined,
};

// Okay.
let myObj2_3: MyInterface2 = {
    myField: null,
};

// Okay.
let myObj2_4: MyInterface2 = {
    myField: "",
};
```